devtools-ci-index.yaml

```diff
@@ -1861,6 +1859,16 @@
             - *qe-osio-dev-cluster-SA-token
     <<: *job_template_defaults
 
+- eclipse-che-automation-template: &eclipse-che-automation-template
```

Comment:
This needs to be moved before you merge.

If you run `jenkins-jobs test` on the yaml, it will give you an error about undefined reference for the template.

templates have to be defined before the jobs that use them.